data-morpher/services/svc-data-mapper/src/main/resources/mappings-legacy/checkmarx/environment.json

```
@@ -0,0 +1,26 @@
+{
+  "top_level_fields": {
+    "_cloudType": "checkmarx",
+    "id": "$.environment_id",
+    "accountid":"$.account_id",
+    "discoverydate": "$._cq_sync_time",
+    "name": "$.domain",
+    "displayName":"$.domain",
+    "groups": "$.groups",
```

Biome (1.9.4)

[error] The key groups was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)